### PR TITLE
tools homework

### PR DESCRIPTION
**Pull request by al14085**

### CODE REVIEW

File: tools/homework3.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+find . -name "test.txt" | xargs grep 'hello'ls ./test.txt | grep "hello"
```
Comment (ohralathe) [inline tools/homework3.sh]: @al14085 Did it match the homework content? btw, Is there any better way?

File: tools/homework5.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+for i in material rxjs libimobiledevice ansible
+do
+cd ~/workspace/$i
+git checkout master
+git pull origin master
+done
```
Comment (ohralathe) [inline tools/homework5.sh]: what is the output of this script? I don't think it can work as before you refactor.

File: tools/homework2.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+shutdown -r now
+
+#create a zip file for folder tmp2
+zip temp2 temp2
+
+#create tar file for folder tmp3
+tar -cvf tmp3 tmp3
+
+#resister linux command（ll）
+alias ll='ls -la'
```
Comment (ohralathe) [inline tools/homework2.sh]: how about 
```
create a file in `~/tmp/test.txt` and put below content inside test.txt by `just 1 command and 1 time`
```?

**Review by ohralathe — CHANGES_REQUESTED** [review]
@al14085 some comments. plz check it

File: tools/homework5.sh (hunk 2 above)
Comment (al14085) [reply tools/homework5.sh]: # input below command in fetch-repos.sh file.
$cd ~/workspace
$cd material
$git checkout master
$git pull origin master

$cd ..    # back to workspace directory
$cd rxjs  # go to rxjs directory
$git checkout master
$git pull origin master

$cd ../libimobiledevice  # go to libimobiledevice directory
$git checkout master
$git pull origin master

$cd ../ansible  # go to libimobiledevice directory
$git checkout master
$git pull origin master
######################################
I executed the above script with the for statement.

File: tools/homework3.sh (hunk 1 above)
Comment (al14085) [reply tools/homework3.sh]: I dont match this homework. I misunderstood. I will modify.

File: tools/homework2.sh (hunk 3 above)
Comment (al14085) [reply tools/homework2.sh]: I forgot printing. I will pull request again after modifying some review.

**Comment by al14085** [comment]
modified homework part 2, 3 and 4

**Comment by ohralathe** [comment]
look better @al14085 . thank you

plz close the PR

**Pull request closed without merging**